parmed/rdkit/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+A package dealing with RDKit integration.
+"""
+
+from parmed.rdkit.rdkit import RDKit
+
+load_rdkit = RDKit.load
+from_smile = RDKit.from_smile
```

Comment:
```from_smiles``` -- it's a SMILES string (the trailing S is part of the acronym).  Does RDKit really use `from_smile`?